CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed no-agent-alert spawn with selected agent in agent-welcome view [#7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029)
 - Fixed security policy exception when it contained deprecated actions [#7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042)
 - Fix export formatted csv data with special characters from tables [#7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048)
+- Fixed the automatic page refresh [#7016](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7016)
```

Comment:
**though**: Could you elaborate where the fix is affecting? It is not clear to me.